### PR TITLE
refactor(autoware-interfaces): use jinja2 template for pages of Autoware interfaces

### PR DESCRIPTION
**Pull request by isamu-takagi**
## Description

Until now some pages were generated using my local scripts. But since other people can't maintain these pages, I rewrote them using templates.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The Reviewers **must** check the checkboxes below before approval.

- [x] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Comment by isamu-takagi** [comment]
Thank you. I'll add readme.

**Comment by isamu-takagi** [comment]
@xmfcx Add readme https://github.com/autowarefoundation/autoware-documentation/pull/372/commits/0a53392471dbf5680ec914e78e177e70ad9647f6. Could you please check?

- `tools/autoware-interfaces/generate.py` needs to be run manually according to readme.
- `main.py` is used by CI.

File: tools/autoware-interfaces/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+## Usage
+
+Create a workspace anywhere and source the dependent message packages.
+
+```bash
+mkdir workspace
+cd workspace
+git clone https://github.com/autowarefoundation/autoware_adapi_msgs.git
+colcon build
+source install/setup.bash
```
Comment (xmfcx) [inline tools/autoware-interfaces/README.md:15]:
```suggestion
mkdir -p workspace/src
cd workspace/src
git clone https://github.com/autowarefoundation/autoware_adapi_msgs.git
cd ..
colcon build
source install/setup.bash
```

Comment (xmfcx) [reply tools/autoware-interfaces/README.md:15]: Should it be like this? Or is your version correct?

Comment (isamu-takagi) [reply tools/autoware-interfaces/README.md:15]: Thanks! It worked without `src` but your version is more preferable.

**Comment by isamu-takagi** [comment]
@xmfcx @mitsudome-r Could you approve this if it looks good?

**Review by xmfcx — APPROVED** [review]

**Pull request merged**